pages/common/git-rev-parse.md

```diff
@@ -0,0 +1,12 @@
+# git rev-parse
+
+> Parses a revision.
```

Comment:
Can we use something more descriptive than "parse"? Maybe something relating to exposing/displaying information (or "metadata") about a revision, WDYT?

Reply:
I agree it's not descriptive enough. The git documentation is even more obscure: `Pick out and massage parameters` (see https://git-scm.com/docs/git-rev-parse)

If you have a better wording, I'm open to suggestions ;)

Reply:
Geez, sometimes it feels like they do it on purpose to make things hard to understand! 😅

Would `Display metadata related to specific revisions` work for you?

Reply:
Sure, see 98a60c0d11b1013b8cb92f0720abb20674fc9b75